tests/rptest/k8s_tests/simple_k8s_test.py

```diff
@@ -20,6 +21,15 @@ def __init__(self, test_context):
         super(SimpleK8sTest, self).__init__(test_context)
         self.redpanda = RedpandaServiceK8s(test_context, 1)
 
+    @property
+    def debug_mode(self):
```

Comment:
Since we are reading an environment variable, and this property is read only from the cluster decorator, do we need this here?

Reply:
Are you suggesting moving the read from the environment check directly into the cluster decorator?

I think there really should probably be some `BaseTest` class that adds this to all our tests.  Thoughts?

Reply:
> Did you try on a debug build and saw greater CPU utilization even when the test was done?

Yeah I added a comment in `cluster`, but a ton of debug build tests where triggering this check, even for tests that seemingly did very little.